internal/policy/root_test.go

```diff
@@ -29,56 +29,84 @@ func TestInitializeRootMetadata(t *testing.T) {
 }
 
-func TestAddTargetsKey(t *testing.T) {
-	keyBytes, err := os.ReadFile(filepath.Join("test-data", "root.pub"))
+func TestAddRootKey(t *testing.T) {
+	key, err := tuf.LoadKeyFromBytes(rootKeyBytes)
 	if err != nil {
 		t.Fatal(err)
 	}
 
-	key, err := tuf.LoadKeyFromBytes(keyBytes)
+	rootMetadata := InitializeRootMetadata(key)
+
+	newRootKey, err := tuf.LoadKeyFromBytes(targets1KeyBytes)
 	if err != nil {
 		t.Fatal(err)
 	}
 
-	rootMetadata := InitializeRootMetadata(key)
+	rootMetadata = AddRootKey(rootMetadata, newRootKey)
+
+	assert.Equal(t, newRootKey, rootMetadata.Keys[newRootKey.KeyID])
+	assert.Equal(t, []string{key.KeyID, newRootKey.KeyID}, rootMetadata.Roles[RootRoleName].KeyIDs)
+}
 
-	targetsKeyBytes, err := os.ReadFile(filepath.Join("test-data", "targets-1.pub"))
+func TestRemoveRootKey(t *testing.T) {
+	key, err := tuf.LoadKeyFromBytes(rootKeyBytes)
 	if err != nil {
 		t.Fatal(err)
 	}
 
-	targetsKey, err := tuf.LoadKeyFromBytes(targetsKeyBytes)
+	rootMetadata := InitializeRootMetadata(key)
+
+	newRootKey, err := tuf.LoadKeyFromBytes(targets1KeyBytes)
 	if err != nil {
 		t.Fatal(err)
 	}
 
-	rootMetadata = AddTargetsKey(rootMetadata, targetsKey)
-	assert.Equal(t, targetsKey, rootMetadata.Keys[targetsKey.KeyID])
-	assert.Equal(t, []string{targetsKey.KeyID}, rootMetadata.Roles[TargetsRoleName].KeyIDs)
+	rootMetadata = AddRootKey(rootMetadata, newRootKey)
+
+	rootMetadata, err = DeleteRootKey(rootMetadata, newRootKey.KeyID)
+
+	assert.Nil(t, err)
+	assert.Equal(t, key, rootMetadata.Keys[key.KeyID])
+	assert.Equal(t, newRootKey, rootMetadata.Keys[newRootKey.KeyID])
+	assert.Equal(t, []string{key.KeyID}, rootMetadata.Roles[RootRoleName].KeyIDs)
+
+	rootMetadata, err = DeleteRootKey(rootMetadata, key.KeyID)
+
+	assert.ErrorIs(t, err, ErrCannotMeetThreshold)
+	assert.Nil(t, rootMetadata)
 }
 
-func TestDeleteTargetsKey(t *testing.T) {
-	keyBytes, err := os.ReadFile(filepath.Join("test-data", "root.pub"))
+func TestAddTargetsKey(t *testing.T) {
+	key, err := tuf.LoadKeyFromBytes(rootKeyBytes)
 	if err != nil {
 		t.Fatal(err)
 	}
 
-	key, err := tuf.LoadKeyFromBytes(keyBytes)
+	rootMetadata := InitializeRootMetadata(key)
+
+	targetsKey, err := tuf.LoadKeyFromBytes(targets1KeyBytes)
 	if err != nil {
 		t.Fatal(err)
 	}
 
-	rootMetadata := InitializeRootMetadata(key)
+	rootMetadata = AddTargetsKey(rootMetadata, targetsKey)
+	assert.Equal(t, targetsKey, rootMetadata.Keys[targetsKey.KeyID])
+	assert.Equal(t, []string{targetsKey.KeyID}, rootMetadata.Roles[TargetsRoleName].KeyIDs)
+}
 
-	targetsKeyBytes, err := os.ReadFile(filepath.Join("test-data", "targets-1.pub"))
+func TestDeleteTargetsKey(t *testing.T) {
+	key, err := tuf.LoadKeyFromBytes(rootKeyBytes)
 	if err != nil {
 		t.Fatal(err)
 	}
 
-	targetsKey1, err := tuf.LoadKeyFromBytes(targetsKeyBytes)
+	rootMetadata := InitializeRootMetadata(key)
+
+	targetsKey1, err := tuf.LoadKeyFromBytes(targets1KeyBytes)
 	if err != nil {
 		t.Fatal(err)
 	}
+	var targetsKeyBytes []byte
```

Comment:
```suggestion
```

I'd just add make it `:=` below instead.

Reply:
Alternatively, that can be added to helpers_test via go:embed too.

Reply:
removed